tools/reviewer/job/sync/GithubClient.java

```diff
@@ -231,21 +260,8 @@ public CommitsResponse getCommits(CommitsRequest request) throws IOException {
     return builder.build();
   }
 
-  public UserResponse getUser(UserRequest request) {
-    UserResponse.Builder builder = UserResponse.newBuilder();
-    String response = doRequest(RequestMethod.GET, String.format(GET_USER, request.getLogin()));
-    try {
-      JsonFormat.parser()
-          .ignoringUnknownFields()
-          .merge(String.format("{\"user\":%s}", response), builder);
-    } catch (IOException e) {
-      e.printStackTrace();
-    }
-    return builder.build();
-  }
-
   private String doRequest(RequestMethod requestMethod, String request) {
-    if (requestMethod.equals(RequestMethod.GET)) {
+    if (requestMethod.equals(RequestMethod.GET) || requestMethod.equals(RequestMethod.DELETE)) {
       return doRequest(requestMethod, request, "");
     } else {
       throw new RuntimeException(requestMethod.name() + " method requires the request data.");
```

Comment:
I think this can be more specific - `s/RuntimeException/IllegalArgumentException`.

Reply:
Done.